Pilot/pilotCommands.py

```diff
@@ -1210,3 +1210,33 @@ def execute( self ):
     """
     self._setNagiosOptions()
     self._runNagiosProbes()
+
+class UnpackDev( CommandBase ):
+  """ Unpack dev.tgz from the pilot directory into the pilot directory
+      Put dev.tgz in the remote pilot directory to have it fetched along
+      with the rest of the pilot scripts. The UnpackDev pilot command
+      needs to be listed after InstallDIRAC if it contains dev versions
+      of DIRAC modules.
+  """
+
+  def __init__( self, pilotParams ):
+    """ c'tor
+    """
+    super( UnpackDev, self ).__init__( pilotParams )
+    self.devFile = 'dev.tgz'
+
+  def execute( self ):
+    """ Standard entry point to a pilot command
+    """
```

Comment:
I'd quite like an error if the command is run but the file isn't supplied. I don't want to be thinking the changes have been applied when they've not.